src/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_DurationScreen/durationPicker/DurationPicker.tsx

```diff
@@ -0,0 +1,252 @@
+import {Platform, View} from 'react-native';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import SvgDate from '@atb/assets/svg/mono-icons/time/Date';
+import {ThemeText} from '@atb/components/text';
+import {TicketAssistantTexts, useTranslation} from '@atb/translations';
+import RNDateTimePicker from '@react-native-community/datetimepicker';
+import {addDays, format} from 'date-fns';
+import {
+  dateDiffInDays,
+  getSliderIndex,
+  getDurationText,
+} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_DurationScreen/utils';
+import {SectionSeparator} from '@atb/components/sections';
+import {Slider} from '@atb/components/slider';
+import React, {useState} from 'react';
+import {StyleSheet} from '@atb/theme';
+import {useAccessibilityContext} from '@atb/AccessibilityContext';
+import {useLocaleContext} from '@atb/LocaleProvider';
+import {useTicketAssistantState} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistantContext';
+import {Button} from '@atb/components/button';
+
+type DurationPickerProps = {
+  duration: number;
+  setDuration: (duration: number) => void;
+};
+const currentDate = new Date();
```

Comment:
It could be the case when this time is used is out or not in synch? it could be used when is really needed.